### DISCUSSION TITLE
Need help with the RetrieveChat example

### DISCUSSION THREAD

**Discussion by techybolek** [microsoft/autogen#328] [Q&A]
Greetings, I need help with the RetrieveChat example please:

https://github.com/microsoft/autogen/blob/main/notebook/agentchat_RetrieveChat.ipynb

The instructions say:

"Navigate to the website folder and run `pydoc-markdown` and it will generate folder `reference` under `website/docs`."

What "website folder" are they talking about? I'm not seeing any. Neither am I seeing 'pydoc-markdown' anywhere.

**Comment by sonichi** [comment]
@thinkall could you help?

**Comment by thinkall** [comment]
For testing with FLAML docs, this is the folder "https://github.com/microsoft/FLAML/tree/main/website", you need to clone the repo to local machine. "pydoc-markdown" is a python package. More details is here: https://github.com/microsoft/FLAML/blob/main/website/README.md.

You can change to any other folder or doc file, no need to bother with this one. @techybolek

**Reply by thinkall** [reply]
To try with autogen docs, here is the folder "https://github.com/microsoft/autogen/tree/main/website"

![image](https://github.com/microsoft/autogen/assets/3197038/902cf3c3-c607-49d4-a337-cfee39692c24)

**Comment by techybolek** [comment]
Thanks. But how do I clone the website folder? What's the correct link?

$ git clone https://github.com/microsoft/FLAML/tree/main/website
Cloning into 'website'...
fatal: repository 'https://github.com/microsoft/FLAML/tree/main/website/' not found

Retrying with permalink:
$ git clone https://github.com/microsoft/FLAML/tree/dc8060a21b1e805f65ef94a0941dd0f5497f1294/website
Cloning into 'website'...
fatal: repository 'https://github.com/microsoft/FLAML/tree/dc8060a21b1e805f65ef94a0941dd0f5497f1294/website/' not found

**Reply by thinkall** [reply]
![image](https://github.com/microsoft/autogen/assets/3197038/4bc03442-4217-4a43-ad2d-fba75eb7c497)
